### PR TITLE
Move verification link

### PR DESCRIPTION
**Pull request by shubhscoder**
This is to fix a typo in #3992 
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] PR is descriptively titled 📑
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

Thanks!

### CODE REVIEW

**Comment by gauravano** [comment]
Hi, just dropping here, how the welcome-email-body looks:

email_feature


Thanks!

**Comment by shubhscoder** [comment]
@gauravano @jywarren is this fine or we need some changes?

File: app/views/welcome_mailer/notify_newcomer.html.erb [hunk 1]
```diff
@@ -18,14 +18,14 @@
...
-    
-    In case you are not able to click the link, try copy pasting the link in the browser.
-
     The Public Lab Community
 <% end %>
 
+Please click on the link below to verify your email.
+
+<% verifaction_link="#{ActionMailer::Base.default_url_options[:host]}/verify/#{@user.generate_token}" %>
+<%= link_to "#{verifaction_link}", verifaction_link%>
```
Comment (gauravano) [inline app/views/welcome_mailer/notify_newcomer.html.erb]: How about changing the name here to `verification_link`?

**Comment by shubhscoder** [comment]
Is it okay now @gauravano ?

**Comment by shubhscoder** [comment]
@jywarren can we merge this?

**Comment by jywarren** [comment]
Great work!!! 🙌

**Pull request merged**